downloads/aws/README.md

```diff
@@ -0,0 +1,81 @@
+Scripted setup to get your aws account ready for Spinnaker
+
+Replaces setup at https://www.spinnaker.io/setup/install/providers/aws/
```

Comment:
Because you reference /setup/install/providers/aws/ in a few places below, I believe this supplements those instructions rather than replaces, correct?

Could this doc go there, instead, and have a different name besides readme? And then just point to the downloadable files?